scripts/teamcity/20.build.sh

```diff
@@ -1,13 +1,23 @@
 #!/usr/bin/env bash
 
-set -eux
+set -eux -o pipefail
+IFS=$'\n\t'
 
 SCRIPT_DIR="$(cd "$(dirname -- "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
 source "$SCRIPT_DIR/../common.sh"
 
 # Fix permission issue when using docker builds
 git config --global --add safe.directory '*'
 
+
+BUILD_TYPE=${BUILD_TYPE:-Debug}
```

Comment:
Any particular reason why this is in there? We only have "release"-type builds for the REST API, I think it can be removed entirely.